code/game/machinery/computer/communications.dm

```diff
@@ -230,6 +230,19 @@ var/const/CALL_SHUTTLE_REASON_LENGTH = 12
 			src.emagged = 0
 			src.updateDialog()
 
+		if("nukerequest") //When there's no other way
+			if(src.authenticated==2)
+				if(CM.cooldownLeft())
+					usr << "Arrays recycling. Please stand by."
+					return
+				var/input = stripped_input(usr, "Please enter the reason for requesting the nuclear self-destruct codes. Misuse of the nuclear request system will not be tolerated under any circumstances.  Transmission does not guarantee a response.", "To abort, send an empty request.","")

```

Comment:
Please remove the ```"To abort, send an empty request."```, this is what the cancel button on the input window is for.
Replace with ```"Request Nuclear Authentication Codes"``` maybe as this is simply what is stated in the title of the input window.